src/components/Filters/Filters.stories.tsx

```diff
@@ -0,0 +1,150 @@
+import { FiltersDrawer } from '../FiltersDrawer/FiltersDrawer';
+
+export default {
+  title: 'Organisms/Tables/Filters',
+  component: Filters,
+  parameters: {
+    badges: [BADGE.BETA],
+  },
+  args: {
+    checkboxFields: [
+      {
+        legend: 'Filters Segment 1',
+        checkboxes: [
+          { label: 'Filters label 1', value: 'Filters label 1' },
+          { label: 'Filters label 2', value: 'Filters label 2' },
+          { label: 'Filters label 3', value: 'Filters label 3' },
+        ],
+      },
+    ],
+    onClose: (checkedValues) => {
+      console.log(checkedValues);
+    },
+  },
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Filters>;
+
+export const DefaultInteractive: StoryObj<Args> = {};
+
+const overflowCheckboxFields = [
+  {
+    legend: 'Filters Segment 1',
+    checkboxes: [
+      {
+        label:
+          'Filters long label 1 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod',
+        value: 'segment1label1',
+      },
+      { label: 'Filters label 2', value: 'segment1label2' },
+      { label: 'Filters label 3', value: 'segment1label3' },
+    ],
+  },
+  {
+    legend: 'Filters Segment 2',
+    checkboxes: [
+      { label: 'Filters label 1', value: 'segment2label1' },
+      {
+        label:
+          'Filters long label 2 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod',
+        value: 'segment2label2',
+      },
+      { label: 'Filters label 3', value: 'segment2label3' },
+    ],
+  },
+  {
+    legend: 'Filters Segment 3',
+    checkboxes: [
+      { label: 'Filters label 1', value: 'segment3label1' },
+      { label: 'Filters label 2', value: 'segment3label2' },
+      {
+        label:
+          'Filters long label 3 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod',
+        value: 'segment3label3',
+      },
+    ],
+  },
+  {
+    legend: 'Filters Segment 4',
+    checkboxes: [
+      { label: 'Filters label 1', value: 'segment4label1' },
+      { label: 'Filters label 2', value: 'segment4label2' },
+      { label: 'Filters label 3', value: 'segment4label3' },
+      { label: 'Filters label 4', value: 'segment4label4' },
+      { label: 'Filters label 5', value: 'segment4label5' },
+      { label: 'Filters label 6', value: 'segment4label6' },
+    ],
+  },
+];
+
+const checkedMap = {};
+overflowCheckboxFields.forEach(({ checkboxes }) => {
+  checkboxes.forEach(({ value }) => {
+    checkedMap[value] = false;
+  });
+});
+
+export const OverflowInteractive: StoryObj<Args> = {
+  args: {
+    checkboxFields: overflowCheckboxFields,
+    onClose: (checkedValues) => {
+      console.log(checkedValues);
+    },
+  },
+};
+
+type FiltersDrawerArgs = React.ComponentProps<typeof FiltersDrawer>;
+/**
+ * 1) Axe testing throws error due to Drawer component not being controlled, but this story is mostly for visual regression testing as the interactive stories would only capture the toggle button.
```

Comment:
should we add a play function so the stories show the open filter?

Reply:
The purpose of the these stories are to snap the open filter (non interactive), and the interactive stories I think should start not closed